easybuild/easyconfigs/s/Sumo/Sumo-1.3.1-GCCcore-8.3.0.eb

```
@@ -0,0 +1,34 @@
+easyblock = 'CMakeMake'
+
+name = 'Sumo'
+version = '1.3.1'
```

@ekieffer please a Python versionsuffix:
```suggestion
version = '1.3.1'
versionsuffix = '-Python-%(pyver)s'
```